src/OpenSSL.jl

```diff
@@ -529,15 +529,15 @@ end
 # Locate oss-modules, which contains legacy shared library
 function _ossl_modules_path()
     @static if Sys.iswindows()
-        bin_dir = dirname(OpenSSL_jll.libssl_path)
+        bin_dir = dirname(OpenSSL_jll.libssl_path::String)
```

Comment:
ideally, this should  be fixed in JLLWrappers to be inferable. I'll look into that.

Reply:
Actually, this is fixed now.

Reply:
Ok, so we don't need this change anymore? How about the one a few lines below?

Reply:
It should be fine to remove both. On 1.9 with JLLWrappers master these should infer to `Union{String, Nothing}`.

Reply:
kk, I'm going to close the PR then. I believe the other infer problem should already be fixed on main